create_hrs_tree.py

```diff
@@ -267,12 +267,13 @@ def get_valid_url(url):
                         if href != '/':
                             split_href = href.split("/")
                             split_url = return_url.split("/")
-                            href_extr = split_href[2]
-                            url_extr = split_url[4]
+                            href_extr = split_href[2] if len(split_href) <= 3 else split_href[3]
+                            url_extr = split_url[5]
```

Comment:
For your information, @terrypalo, I had to change the index number to reflect the additional URL depth on the source website. This should not apply when executing `hrscurrent`.